apcd-cms/src/apps/registrations/templates/submission_form/registration_form_body.html

```diff
@@ -345,25 +346,25 @@ <h6>File Submission</h6>
 
                     <ul id="types_of_files" class="checkboxselectmultiple">
                         <li>
-                            <label for="types_of_files_eligibility_enrollment_{{forloop.counter}}_{{r.reg_id}}"
+                            <label for="types_of_files_eligibility_enrollment_{{ent_no}}_{{r.reg_id}}"
                             ><input
                                 type="checkbox"
-                                name="types_of_files_eligibility_enrollment_{{forloop.counter}}_{{r.reg_id}}"
+                                name="types_of_files_eligibility_enrollment_{{ent_no}}_{{r.reg_id}}"
                                 required
-                                id="types_of_files_eligibility_enrollment_{{forloop.counter}}_{{r.reg_id}}"
+                                id="types_of_files_eligibility_enrollment_{{ent_no}}_{{r.reg_id}}"
                                 checked=""
                                 disabled
                             />Eligibility/Enrollment*</label
                             >
                         </li>
-                        {% for file_type, file_type_selected in r.view_modal_content.entities.files_type.items %}
+                        {% for file_type, file_type_selected in entity.files_type.items %}
                         {% if not forloop.first %}
                         <li>
-                            <label for="types_of_files_{{ file_type|lower }}_{{forloop.counter}}_{{r.reg_id}}"
+                            <label for="types_of_files_{{ file_type|lower }}_{{ent_no}}_{{r.reg_id}}"
                                 ><input
                                 type="checkbox"
-                                name="types_of_files_{{ file_type|lower }}_{{forloop.counter}}_{{r.reg_id}}"
-                                id="types_of_files_{{ file_type|lower }}_{{forloop.counter}}_{{r.reg_id}}"
+                                name="types_of_files_{{ file_type|lower }}_{{ent_no}}_{{r.reg_id}}"
```

Comment:
Should this file type field be required for each entity? Ignore this comment if not.

Reply:
I'll take the (required) part out of this section, but UTH does want those checkboxes on each entity (documenting our slack conversation about this here for relevance).

Reply:
Done.